src/screens/HSwitchRemoteFilter.res

```diff
@@ -144,13 +146,26 @@ module RemoteTableFilters = {
     open APIUtils
 
     let (filterDataJson, setFilterDataJson) = React.useState(_ => None)
-
+    let updateDetails = useUpdateMethod()
+    let defaultDate = getDateFilteredObject(~range=30, ())
+    let start_time = filterValueJson->getString(startTimeFilterKey, defaultDate.start_time)
+    let end_time = filterValueJson->getString(endTimeFilterKey, defaultDate.end_time)
     let fetchDetails = useGetMethod()
 
     let fetchAllFilters = async () => {
       try {
         setFilterDataJson(_ => None)
-        let response = await fetchDetails(filterUrlV2)
+        let response = switch apiType {
+        | Post => {
+            let body =
+              [
+                (startTimeFilterKey, start_time->JSON.Encode.string),
+                (endTimeFilterKey, end_time->JSON.Encode.string),
+              ]->Dict.fromArray
```

Comment:
Can directly use `getJsonFromArrayOfJson` instead of `Dict.fromArray->JSON.Encode.object`